src/components/Select/Select.tsx

```diff
@@ -10,25 +10,28 @@ import './Select.scss';
 
 export type SelectableOption = { value: string; label: string; disabled?: boolean };
 
-export type SelectProps = Omit<React.HTMLAttributes<HTMLDivElement>, 'value' | 'onChange'> & {
+export type SelectProps = Omit<
+  React.HTMLAttributes<HTMLDivElement>,
+  'defaultValue' | 'value' | 'onChange'
```

Comment:
Is it required to omit these? AFAIK these are not on `React.HTMLAttributes<HTMLDivElement>`.

Reply:
well, if not, onChange would look like

(property) onChange?: (((event: React.FormEvent<HTMLDivElement>) => void) & ((value: string) => void)) | undefined

Reply:
but now it's like

`(JSX attribute) onChange?: ((value: string) => void) | undefined`

Reply:
I see, but should `value` and `defaultValue` be omitted too?

Reply:
yes, we would like to limit those from

`defaultValue?: string | number | string[];`

to `string | undefined`

Reply:
ok, it's weird that `React.HTMLAttributes<HTMLDivElement>` has those